test/e2e/federated-namespace.go

```diff
@@ -97,7 +94,23 @@ var _ = framework.KubeDescribe("Federation namespace [Feature:Federation]", func
 	})
 })
 
-func deleteAllTestNamespaces(clientset *federation_release_1_4.Clientset) {
+func deleteAllTestNamespacesFed14(clientset *federation_release_1_4.Clientset) {
```

Comment:
I'm wondering whether instead of duplication code, would it be possible to pass:
"NamespacerInterface"
as an argument.

Then it should be possible to call it with both:
- f.FederationClientset_1_4.Core().Namespaces()
- clientset.Core().Namespaces()